package.json

```diff
@@ -8,54 +8,48 @@
     "registry": "https://registry.npmjs.org"
   },
   "dependencies": {
-    "lerna": "^6.0.3"
-  },
-  "devDependencies": {
     "@types/ioredis": "^4.0.13",
-    "@types/jest": "^27.4.1",
+    "@types/jest": "^29.2.3",
     "@types/jsonpath": "^0.2.0",
-    "@types/node": "^12.6.3",
+    "@types/node": "^18.0.0",
     "@types/pegjs": "^0.10.1",
-    "@types/prettier": "^1.16.4",
-    "coveralls": "^3.0.5",
+    "@types/prettier": "^1.18.2",
+    "coveralls": "^3.0.6",
     "glob": "^7.1.6",
-    "jest": "^27.5.1",
-    "jest-environment-node-debug": "^2.0.0",
-    "jest-extended": "^2.0.0",
+    "jest": "^29.3.1",
+    "jest-extended": "^3.2.0",
+    "lerna": "^6.0.3",
     "pegjs": "^0.10.0",
-    "prettier": "^2.1.2",
-    "rimraf": "^2.6.3",
-    "sqrl": "file:packages/sqrl",
-    "sqrl-cli": "file:packages/sqrl-cli",
-    "sqrl-cli-functions": "file:packages/sqrl-cli-functions",
-    "sqrl-common": "file:packages/sqrl-common",
-    "sqrl-example-functions": "file:examples/sqrl-example-functions",
-    "sqrl-jsonpath": "file:packages/sqrl-jsonpath",
-    "sqrl-load-functions": "file:packages/sqrl-load-functions",
-    "sqrl-redis-functions": "file:packages/sqrl-redis-functions",
-    "sqrl-test-utils": "file:packages/sqrl-test-utils",
-    "sqrl-text-functions": "file:packages/sqrl-text-functions",
-    "ts-jest": "^27.1.3",
-    "ts-node": "^8.3.0",
-    "ts-pegjs": "^0.2.5",
-    "tslint": "^5.18.0",
-    "tslint-config-prettier": "^1.18.0",
+    "prettier": "^2.7.1",
+    "rimraf": "^3.0.0",
+    "ts-jest": "^29.0.3",
+    "ts-node": "^10.9.1",
+    "ts-pegjs": "^0.2.6",
     "typedoc": "^0.23.20",
-    "typescript": "^4.8.4",
-    "websqrl": "file:websqrl",
-    "wikipedia-diff-stream": "file:tools/wikipedia-diff-stream"
+    "typescript": "~4.8.4"
+  },
+  "workspaces": [
+    "examples/sqrl-example-functions",
+    "packages/*",
+    "tools/wikipedia-diff-stream",
+    "website",
+    "websqrl"
```

Comment:
The `lerna publish` command tries to publish new versions of these... which is why they weren't listed in `workspaces` before. I don't know what the best solution here is, maybe it's not a problem with the new `npm`.

Reply:
ahhh gotcha. Listing them is best, so the package deps get installed when we run `npm i` at the repository root. Lerna should theoretically skip private packages I’ll mark `wikipedia-diff-stream` as private as well so it gets ignored as well.